app/src/Html.idr

```diff
@@ -28,10 +28,10 @@ logAndDisplay e       s = clearMsg e <+> displaySketcher "app" e s
 covering export
 app : IO ()
 app =
-  let se := defaultSettings abbreviations
+  let se := defaultSettings abbreviations Nothing
    in runMVC
         update
         (logAndDisplay @{se})
         (putStrLn . dispErr)
         (KeyDown "Escape")
-        (init @{se} (SD 600 400) Init "")
+        (init @{se} (SD 400 266) Init "")
```

Comment:
Why this adjustment? If we use the app as a GitHub page, we probably should make the initial size of the drawing window dependent on the extension we use.